var/spack/repos/builtin/packages/libelf/package.py

```diff
@@ -29,8 +29,23 @@ class Libelf(AutotoolsPackage):
 
     provides("elf@0")
 
+    # configure: error: neither int nor long is 32-bit
+    depends_on("automake", when="platform=darwin target=aarch64:", type="build")
+    depends_on("autoconf", when="platform=darwin target=aarch64:", type="build")
+    depends_on("libtool", when="platform=darwin target=aarch64:", type="build")
+    depends_on("m4", when="platform=darwin target=aarch64:", type="build")
+
+    @property
+    def force_autoreconf(self):
+        return self.spec.satisfies("platform=darwin target=aarch64:")
+
     def configure_args(self):
-        args = ["--enable-shared", "--disable-dependency-tracking", "--disable-debug"]
+        args = ["--enable-shared", "--disable-debug"]
```

Comment:
The `--disable-dependency-tracking` flag is not recognized by configure

Reply:
Yep, this option should never be needed, it's only a shortcut.